### PR TITLE
Use "bar" instead of "tact"

### LINKED ISSUE #4865: Use "bar" instead of "tact"
I have never seen the term "tact" referring to a bar anywhere except LMMS's source code, but I have found the reason for this: confusion with German _Takt_ (https://www.reddit.com/r/explainlikeimfive/comments/7bmitw/eli5_difference_between_beat_bar_steps_tempo_tact/)

I suggest changing the term from "tact" to "bar" before even more people reading the code get confused.

### PR DESCRIPTION
**Pull request by M374LX**
Closes #4865.

### CODE REVIEW

**Comment by qnebra** [comment]
I think in code is unnecessary to change from tact to bar. Only in user space maybe.

**Comment by BaraMGB** [comment]
In my opinion this is a good idea. Helps new people to understand what's going on.

**Comment by tresf** [comment]
> I think in code is unnecessary to change from tact to bar. Only in user space maybe.

No, that would be worse.  The word `tact` is a common complaint amongst new developers; this is an improvement.

There are a few unnecessary whitespace additions/subtractions, but they're in files that are already inconsistent.  This looks good to merge.  I'm not as actively involved with the codebase/open PRs as a few others.  @PhysSong would you be comfortable merging this?

**Comment by PhysSong** [comment]
> @PhysSong would you be comfortable merging this?

No objections.

**Comment by JohannesLorenz** [comment]
If tresf and PhysSong have no objections, I'll consider this as accepted.

I'll start a review.

File: src/core/Track.cpp [hunk 1]
```diff
@@ -2438,17 +2437,17 @@ void Track::createTCOsForBB( int bb )
  *    in ascending order by TCO time, once we hit a TCO that was earlier
  *    than the insert time, we could fall out of the loop early.
  */
-void Track::insertTact( const MidiTime & pos )
+void Track::insertBar( const MidiTime & pos )
 {
 	// we'll increase the position of every TCO, positioned behind pos, by
-	// one tact
+	// one bar
```
Comment (JohannesLorenz) [inline src/core/Track.cpp]:
```suggestion
	// one bar
```

File: src/core/Track.cpp [hunk 2]
```diff
@@ -1178,13 +1178,12 @@ TrackContentWidget::~TrackContentWidget()
...
-	const int tactsPerBar = 4;
 	const TrackContainerView * tcv = m_trackView->trackContainerView();
 
-	// Assume even-pixels-per-tact. Makes sense, should be like this anyways
-	int ppt = static_cast<int>( tcv->pixelsPerTact() );
+	// Assume even-pixels-per-bar. Makes sense, should be like this anyways
+	int ppb = static_cast<int>( tcv->pixelsPerBar() );
 
-	int w = ppt * tactsPerBar;
+	int w = ppb * 4;
```
[the remark this answers is not in the document]
Comment (Spekular) [reply src/core/Track.cpp]: Is it actually correct? Couldnt tactsPerBar change depending on the time signature?

[the remark this answers is not in the document]
Comment (JohannesLorenz) [reply src/core/Track.cpp]: I don't even get what `tactsPerBar` means. A tact is a bar, so it should always be `1` :smile: But it's `4`, so that really can't be what they meant...?

[the remark this answers is not in the document]
Comment (PhysSong) [reply src/core/Track.cpp]: I guess it's about the background color.
image
Here 4 bars form a group. It looks weird with the time signature changed, e.g. 5/3, but it's out of scope.

**Comment by JohannesLorenz** [comment]
BUMP @M374LX :smiley:

File: src/core/Track.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (M374LX) [reply src/core/Track.cpp]: Indeed, it is about the background color.

**Comment by PhysSong** [comment]
@JohannesLorenz Can you see if your comments are addressed properly?

File: src/core/Track.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (Spekular) [reply src/core/Track.cpp]: Then can you rename the constant to something like barsPerGroup and leave a comment explaining what it's there for? I feel like the current change makes things harder for someone who wants to adjust these regions in the future.

[the remark this answers is not in the document]
Comment (M374LX) [reply src/core/Track.cpp]: @Spekular 
Done.

**Comment by lukas-w** [comment]
I assume the last open request https://github.com/LMMS/lmms/pull/4878#discussion_r281038468 by @JohannesLorenz was resolved by 16c760ceb49f321f8f1fbd2edc03a3c294a2f907. I'll squash some commits and merge this.

**Comment by lukas-w** [comment]
Merged via 53e6b645c8e550156aa312a1b8b0d4b085bf7eac and 46f5433732654d8090cc75ee21f3a00f93910439,
